src/status_im/contexts/communities/actions/share_community/view.cljs

```diff
@@ -0,0 +1,92 @@
+(ns status-im.contexts.communities.actions.share-community.view
+  (:require
+    [quo.core :as quo]
+    [quo.foundations.colors :as colors]
+    [react-native.core :as rn]
+    [react-native.fast-image :as fast-image]
+    [react-native.platform :as platform]
+    [react-native.safe-area :as safe-area]
+    [status-im.common.qr-codes.view :as qr-codes]
+    [status-im.common.resources :as resources]
+    [status-im.contexts.communities.actions.share-community.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn view
+  []
+  (let [{:keys [url community-id]} (rf/sub [:get-screen-params])
+        window-width               (rf/sub [:dimensions/window-width])]
+    (fn []
```

Comment:
render function shouldn't be used anymore

Reply:
Fixed.